modules/createTransitionManager.js

```diff
@@ -104,7 +104,7 @@ export default function createTransitionManager(history, routes) {
     return route.__id__ || create && (route.__id__ = RouteGuid++)
   }
 
-  const RouteHooks = {}
+  const RouteHooks = Object.create(null)
```

Comment:
This is an internal-only object used as a map, so it's more correct to use a `null`-prototype object than a full-fledged object as a map. There's no concern with breaking any consumers, as `RouteHooks` is not visible outside this module.